mcs/build/profiles/mobile.make

```diff
@@ -16,7 +16,6 @@ DEFAULT_REFERENCES = -r:$(topdir)/class/lib/$(PROFILE)/mscorlib.dll
 PROFILE_MCS_FLAGS = \
 	-d:NET_1_1 \
 	-d:NET_2_0 \
-	-d:NET_2_1 \
```

Comment:
We should probably leave this in, just as a safeguard if these defines are used somewhere else.